package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "desktop-notifications",
-  "version": "0.1.7",
+  "version": "0.2.0",
```

Comment:
I will revert this before merging and manage the release separately. I also want to enable CI for macOS before doing the release, of course, but in a separate PR 😳